### PR TITLE
feat(step2): template for step 2

### PR DESCRIPTION
**Pull request by annawen1**
### Description

Template for step 2 - still needs some work but will address later

### Changelog

**New**

- {{new thing}}

**Changed**

- {{changed thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,68 +1,228 @@
...
+    <dds-dotcom-shell-container>
+      <div class="bx--grid">
+    <div class="bx--row">
+      <div class="bx--col-lg-8 bx--offset-lg-4 bx--no-gutter">
+        <dds-leadspace-with-search adjacent-theme="g90-and-g100">
+          <dds-leadspace-block-heading slot='heading'>Find analytics services</dds-leadspace-block-heading>
+          <dds-leadspace-block-content slot='content'>
+            <dds-leadspace-search-block-heading>Make fast and accurate data-based decisions</dds-leadspace-search-block-heading>
+          </dds-leadspace-block-content>
+          <dds-search-with-typeahead slot='search' leadspace-search><dds-search-with-typeahead/>
```
Comment (ariellalgilmore) [inline src/index.html:83]: small thing
```suggestion
          <dds-search-with-typeahead slot='search' leadspace-search></dds-search-with-typeahead>
```

File: src/assets/scss/index.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+// Tutorial styles will go here
```
Comment (ariellalgilmore) [inline src/assets/scss/index.scss:1]: Should we remove this?

Comment (annawen1) [reply src/assets/scss/index.scss:1]: yup let me delete that

**Review by ariellalgilmore — APPROVED** [review]
AMAZINGGGG!!! Thanks @annawen1

**Pull request merged**